librz/il/il_opcodes.c

```diff
@@ -205,6 +232,26 @@ RZ_API RZ_OWN RzILOpBool *rz_il_op_new_bitv_from_st64(ut32 length, st64 number)
 	return ret;
 }
 
+/**
+ * \brief Creates a new bitvector from the bits \p n to \p m of src bitvector \p v ('s bitv -> 'a bitv).
+ *
+ * \param v Src bitvector
+ * \param n Start index (counted from LSB).
+ * \param m End index (counted from LSB).
+ * \return Bitvector with bits m:n from \p v.
+ */
+RZ_API RZ_OWN RzILOpBool *rz_il_op_new_bitv_from_bitv(RZ_NONNULL RZ_BORROW RzILOpBitVector *v, ut32 m, ut32 n) {
```

Comment:
I think it's better to call it `rz_il_op_bitv_slice()`. @thestr4ng3r what do you think?

Reply:
It should be called `extract` and defined exactly like http://binaryanalysisplatform.github.io/bap/api/master/bap-core-theory/Bap_core_theory/Theory/module-type-Basic/index.html#val-extract, only with the final cast to sort `s` omitted.

Reply:
This was an artifact from my confusion. Removed it since it wasn't used anyways.